recipes/pixman/all/patches/0003-Adjustments-to-build-with-llvm.patch

```diff
@@ -0,0 +1,3211 @@
+diff --git a/pixman/pixman-arm-asm.h b/pixman/pixman-arm-asm.h
```

Comment:
wow, this path file is incredibly huge. I think this will be a burden if we ever have to tune it or even change some of its lines. The only positive thing is that it is well-referenced with a link to its origin in the conandata, but I still have my hesitations to add it here 😢

Reply:
yeah, let's hope https://gitlab.freedesktop.org/pixman/pixman/-/merge_requests/71 will be merged as-is, so that we can drop the patch with the next version (and drop 0.42.2). The alternative is to disable macos for this version of pixman.

Reply:
yeah, I think it is too much right now. Let's wait for the upstream fix

Reply:
ok. FTR, it could take a long time, seeing the release history https://gitlab.freedesktop.org/pixman/pixman/-/tags